inst/templates/bioc-support.md

```diff
@@ -5,7 +5,7 @@ Before filing an issue, there are a few places to explore and pieces to put toge
 
 ## Make a reprex
 
-Start by making a minimal **repr**oducible **ex**ample using the  [reprex](https://reprex.tidyverse.org/) package.
+Start by making a minimal **repr**oducible **ex**ample using the [reprex](https://reprex.tidyverse.org/) package.
 If you haven't heard of or used reprex before, you're in for a treat!
 Seriously, reprex will make all of your R-question-asking endeavors easier (which is a pretty insane ROI for the five to ten minutes it'll take you to learn what it's all about).
 For additional reprex pointers, check out the [Get help!](https://www.tidyverse.org/help/) section of the tidyverse site.
```

Comment:
I liked the pointer to StackOverflow in the earlier document, rather than tidyverse, since we are not talking about tidyverse packages here...

Also in the first line of the 'Where to ask?' section, I don't think we should be pointing to tidyverse -- if bioconductor.org needs updating, or if there is already a better place on bioconductor.org, then let's point there instead.

Also, in the 'If it's a question...' bullet, I don't know why one would look to community.rstudio.com as a first stop. It's a Bioconductor package so support.bioconoductor.org and then if not perhaps it's a general R question so StackOverflow? and finally rstudio?